config/i18next-scanner.config.js

```diff
@@ -0,0 +1,62 @@
+const fs = require('fs');
+const chalk = require('chalk');
+const path = require('path');
+
+// fallback language
+let defaultContent = {};
+try {
+  defaultContent = JSON.parse(
+    fs.readFileSync(
+      path.resolve(__dirname, '../locales/en/base.json'),
+      'utf8'
+    ) || {}
+  );
+} catch (err) {
+  console.log('default langauge not found');
+}
+
+module.exports = {
+  options: {
+    debug: true,
+    func: {
+      list: ['i18next.t', 'i18n.t', 't']
+    },
+    lngs: ['en', 'de', 'fr', 'ru', 'es', 'it', 'fa'],
```

Comment:
So there's languages specified (hardcoded) also in `src/shared/i18n/index.js`.. Would it be easiest to store them both somewhere together and use them in both places?